modules/zip/docs/README.md

```diff
@@ -11,3 +11,5 @@ npm install @loaders.gl/core @loaders.gl/zip
 ## Attributions
 
 ZipLoader is a wrapper around the [JSZip module](https://stuk.github.io/jszip/). JSZip has extensive documentation on options (and more functionality than this loader object can expose).
+
+TarBuilder uses a modified version of [tar-js](https://github.com/beatgammit/tar-js), which is under MIT license, for tar archive construction.
```

Comment:
Nit: "uses a modified version" => "is a fork of"   (the former sounds like it might not be in this repo).